feat: CI/CD CI/CD for running test and deploying package to GPR

djolewalker opened the pull request:
by: https://github.com/evaj10

closes #9

evaj10 reviewed (commented):
In order to build nuget package, the following needs to be added to 
https://github.com/Vegari-1/MessageService/blob/dev/MessageService/MessageService/MessageService.csproj file:

Within `<PropertyGroup>` tag:
   ```
 <IsPackable>true</IsPackable>
 <RepositoryUrl>https://github.com/Vegari-1/MessageService</RepositoryUrl>
```

djolewalker commented:
> In order to build nuget package, the following needs to be added to https://github.com/Vegari-1/MessageService/blob/dev/MessageService/MessageService/MessageService.csproj file:
> 
> Within `<PropertyGroup>` tag:
> 
> ```
> <IsPackable>true</IsPackable>
> <RepositoryUrl>https://github.com/Vegari-1/MessageService</RepositoryUrl>
> ```

Requested packaging configuration is added.

djolewalker committed:
feat: CI/CD CI/CD for running test and deploying package to GPR

by: https://github.com/evaj10

closes #9
Packaging configuration


Prerelease configuration


fix: use dev instead develop branch in actions
